.env.example

```diff
@@ -35,4 +35,5 @@ USER_TOKEN_SECRET=secret
 ONE_LOGIN_ENABLED=true
 SESSION_COOKIE_NAME=session_id
 ADMIN_BACKEND_HOST=http:localhost:8081
-V2_LOGOUT_URL=http://localhost:8082/v2/logout
\ No newline at end of file
+V2_LOGOUT_URL=http://localhost:8082/v2/logout
+APPLICANT_BACKEND_HOST=http://localhost:8080
```

Comment:
is there an infra pr for this change

Reply:
Yeah, that one's with Dylan though, I don't have the skills or the AWS access unfortunately lol